src/api/api.js

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] I would not recommend disabling eslint. instead, try to make the code standard to follow the eslint rules. :slightly_smiling_face:

Reply:
Ok. I found a way to solve it. Thanks for pointing it out :+1: